lang/go/code-types.talon-list

```diff
@@ -0,0 +1,36 @@
+list: user.code_type
+code.language: go
+-
+
+bool: bool
+boolean: bool
+
+byte: byte
+bite: byte
+
+complex sixty four: complex64
+complex six four: complex64
+complex one twenty eight: complex128
+complex one two eight: complex128
+
+number: int
+integer: int
+int: int
+integer eight: int8
+integer sixteen: int16
+integer thirty two: int32
+integer sixty four: int64

```

Comment:
I believe these are more commonly pronounced `int sixty four`. Here's most of my types talon-list:

```
int sixty four: int64
int sixty four: int64
you int sixty four: uint64
int thirty two: int32
you int thirty two: uint32
int sixteen: int16
you int sixteen: uint16
int eight: int8
you int eight: uint8
float thirty two: float32
float sixty four: float64
complex sixty four: complex64
complex one twenty eight: complex128
you int pointer: uintptr
you int: uint
```